linux/debian/libkmnkbp0-0.symbols

```diff
@@ -1,43 +1,43 @@
-libkmnkbp0.so.0 libkmnkbp0-0 #MINVER#
+libkeymancore.so.1 libkmnkbp0-0 #MINVER#
```

Comment:
Is this still supposed to be libkmnkbp0-0 here? And noting the filename is still given as libkmnkbp0-0.symbols.